recipes/cpputest/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class CppUTestConan(ConanFile):
+    name = "cpputest"
+    description = \
+"CppUTest is a C /C++ based unit xUnit test framework for unit testing and for test-driving your code." \
+"It is written in C++ but is used in C and C++ projects and frequently" \
+"used in embedded systems but it works for any C/C++ project."
+    license = "BSD-3-Clause"
+    topics = ("conan", "testing", "unit-testing")
+    homepage = "https://cpputest.github.io"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "use_std_c_lib": ["ON", "OFF"],
+        "use_std_cpp_lib": ["ON", "OFF"],
+        "use_cpp11": ["ON", "OFF"],
+        "detect_mem_leaks": ["ON", "OFF"],
+        "extensions": ["ON", "OFF"],
+        "longlong": ["ON", "OFF"],
+    }
+    default_options = {
+        "fPIC": True,
+        "use_std_c_lib": "ON",
+        "use_std_cpp_lib": "ON",
+        "use_cpp11": "ON",
+        "detect_mem_leaks": "ON",
+        "extensions": "ON",
+        "longlong": "ON",
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if not self.options.use_std_cpp_lib:
+            del self.settings.compiler.libcxx
+            del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def build(self):
+        # tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+        #                       "set (CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
```

Comment:
```suggestion
```

Not used

Reply:
Thanks - to minimise the number of CI builds triggered, I've made that manually on my local build.